packages/terra-table/src/index.js

```diff
@@ -1,5 +1,12 @@
 import Table from './Table';
 import GridContext, { GridConstants } from './utils/GridContext';
+import cellShape from './proptypes/cellShape';
+import columnShape from './proptypes/columnShape';
+import rowShape from './proptypes/rowShape';
+import sectionShape from './proptypes/sectionShape';
+import validateRowHeaderIndex from './proptypes/validators';
 
 export default Table;
-export { GridContext, GridConstants };
+export {
+  GridContext, GridConstants, cellShape, columnShape, rowShape, sectionShape, validateRowHeaderIndex,
```

Comment:
Do we want to give consumers access to these exports? If these are only used internally within Terra then maybe we can import them using the full path as an exception.

Reply:
Would it hurt for them to be exported?  I was trying to keep them encapsulated but also reuse the logic as we previously had duplicates.

Reply:
Will table (🥁) this topic for now so approving the PR.